docs/reference/data-collection/data-collection.md

```diff
@@ -0,0 +1,100 @@
+---
+id: data-collection
+title: "Data collection"
+description: "Learn about the data collected by Camunda."
+---
+
+Camunda collects a variety of data to evaluate contractual usage, provide a better user experience, and improve its products. This section describes which data is collected, how Camunda ensures privacy, and what options you have to modify which data is sent to Camunda. This document applies only to interactions with Camunda's products.
+
+## Principles
+
+Camunda follows certain principles to ensure the privacy of its customers and the success of its product development efforts.
+
+- Camunda minimizes automatic collection of personally identifiable information (PII) to information required to identify the user like name and email. Where PII is collected, customers have the option to opt-out of sharing this data with Camunda.
+- Camunda avoids automatic collection of protected health information (PHI).
+  - Data collected from end-users such as form fills or process variables are not tracked.
+- Camunda avoids automatic collection of intellectual property (IP).
+  - Assets describing how a process is defined and executed are not automatically shared with Camunda.
+- Camunda does not collect payment information nor sell user information.
+
+## Data Collection
+
+Camunda collects data through a variety of methods.
+
+### Contractual Usage
+
+Camunda collects a limited set of contractually-agreed [usage metrics](/reference/data-collection/usage-metrics.md) to evaluate billing. These are summary metrics that contain no sensitive information.
+
+### Environment Data
+
+Camunda collects information about customer installations to enable better support and product improvement decisions.
+
+Self-Managed customers may choose to send a very limited set of information through the Console component's telemetry mechanism. View the [telemetry](/self-managed/console-deployment/telemetry.md) page for an example of this data.
+
+### Usage Data
+
+Camunda collects product usage data from SaaS and Desktop Modeler users to help make better product improvement decisions and enable outreach to support users.
+
+- Feature Usage:
+  - SaaS System Actions: All SaaS organizations submit basic information about which features are being used. When certain features are used, Camunda logs which feature is used and basic information about how it has been used. This information is tied to an pseudonymized organization.
+  - SaaS User Actions: Users that opt-in to analytics cookies
```

Comment:
@conceptualshark can you please include screenshots of these 2 parts of the UI to explain how to opt-out of analytics cookies?
![image](https://github.com/user-attachments/assets/982767aa-a307-4b91-814d-b9a3e70642e0)
![image](https://github.com/user-attachments/assets/2a55b820-15aa-4391-accf-da25d05bf53f)